class.jetpack-gutenberg.php

```diff
@@ -87,6 +85,20 @@ private static function prepend_block_prefix( $block_name ) {
 		return 'jetpack/' . $block_name;
 	}
 
+	/**
+	 * Remove the 'jetpack/' or jetpack-' prefix from an extension name
```

Comment:
Just leaving here [my previous comment](https://github.com/Automattic/jetpack/pull/11212#discussion_r254288087) so it is not lost in the outdated comments:

> According to the code this is removing only the `jetpack/` prefix, so it seems we're not considering the plugins here (unless I'm missing something obvious).
>
> Should we then update the function to handle both of them?
>
> ```php
> private static function remove_extension_prefix( $extension_name ) {
> 	return preg_replace( '/jetpack[\/-]/', '', $extension_name);
> }
>```

Reply:
Thanks Miguel, you're right -- just editing the JSDoc didn't cut it, sorry about that :sweat_smile: 
Pushed b3d9ff484 to fix properly.